os/arch/Kconfig

```diff
@@ -15,13 +15,20 @@ config ARCH_ARM
 	select ARCH_HAVE_CUSTOMOPT
 	---help---
 		The ARM architectures
+config ARCH_XTENSA
```

Comment:
1. Please make an empty line from another config block.
2. Is Xtensa a name of cpu family?

Reply:
Fixed, thanks

Reply:
Fixed, Thanks.
Yes, Xtensa is the Cadence Tensilica Xtensa actictures.